### PR TITLE
Add KleeKai (KLEE) (ERC20)

### PR DESCRIPTION
**Pull request by CryptoCirca**
Adding info for ERC20 token 'KleeKai (KLEE)'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by Cryptocool1** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 10,000 holder addresses and 15,000 transactions, in order to filter out scam tokens.
See the criteria at : https://developer.trustwallet.com/assets/requirements .

**Comment by CryptoCirca** [comment]
Hi,

Our V1 project has 27,000 holders and our V2 project is approaching 4,000 holders.

We are going through a token migration and moving holders over, please review and know that we are a year old project now and not a "scammy" token that was launched a short time ago.

Link to migration information below:

https://kleekaitoken.medium.com/kleekai-migration-all-you-need-to-know-39a3a60b36a6

**Comment by Cryptocool1** [comment]
hello, what is your old contract address?

**Comment by Cryptocool1** [comment]
hello, what is your old contract address?

**Comment by Cryptocool1** [comment]
Please change the description of the token

**Comment by CryptoCirca** [comment]
> Please change the description of the token

Hi, old CA is 

https://etherscan.io/token/0x382f0160c24f5c515a19f155bac14d479433a407

File: blockchains/ethereum/assets/0xA67E9F021B9d208F7e3365B2A155E3C55B27de71/info.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "KleeKai",
+    "type": "ERC20",
+    "symbol": "KLEE",
+    "decimals": 9,
+    "website": "https://kleerun.game",
+    "description": "Token requires updating of the Logo",
```
Comment (Cryptocool1) [inline blockchains/ethereum/assets/0xA67E9F021B9d208F7e3365B2A155E3C55B27de71/info.json:7]: should be description of the token

**Review by CryptoCirca — COMMENTED** [review]
{
    "name": "KleeKai",
    "type": "ERC20",
    "symbol": "KLEE",
    "decimals": 9,
    "website": "https://kleerun.game",
    "description": "This token sports a Play-2-Earn Cryptocurrency game namely called "KleeRun". This game is an addictive, fast paced running game and fun for all ages! Available on Google Play, PC & Apple IOS devices.",
     "explorer": "https://etherscan.io/token/0xA67E9F021B9d208F7e3365B2A155E3C55B27de71",
    "status": "active",
    "id": "0xA67E9F021B9d208F7e3365B2A155E3C55B27de71",
    "links": [
        {
            "name": "twitter",
            "url": "https://twitter.com/KleeKaiETH"
        },
        {
            "name": "telegram",
            "url": "https://t.me/kleeentryportal"
        },
        {
            "name": "discord",
            "url": "https://discord.com/invite/kleekai"
        },
        {
            "name": "coinmarketcap",
            "url": "https://coinmarketcap.com/currencies/kleekai/"
        },
        {
            "name": "coingecko",
            "url": "https://coingecko.com/en/coins/kleekai"
        }
    ],
    "tags": [
        "gamefi"
    ]
}

**Comment by Cryptocool1** [comment]
"Token requires updating of the Logo", -- that is not the description of the token

**Comment by Cryptocool1** [comment]
should be few sentence summary of the token/project

**Comment by CryptoCirca** [comment]
> should be few sentence summary of the token/project

Fixed, thank you for the clarification

**Comment by Cryptocool1** [comment]
the description still the same, please check

**Comment by Cryptocool1** [comment]
submission looks good, awaiting for fee payment

**Comment by CryptoCirca** [comment]
> submission looks good, awaiting for fee payment

Trying to swap BEP20 TWT to BEP2 TWT however wallet isn't set on Multicoin - Will be sorted in a few moments.

**Comment by CryptoCirca** [comment]
Hi,

Our V1 project has 27,000 holders and our V2 project is approaching 4,000
holders.

We are going through a token migration and moving holders over, please
review and know that we are a year old project now and not a "scammy" token
that was launched a short time ago.

Link to migration information below:

https://kleekaitoken.medium.com/kleekai-migration-all-you-need-to-know-39a3a60b36a6

**Pull request merged**